### PR TITLE
Side menu fix

### PR DESCRIPTION
**Pull request by skyrowl**
Fix of sidebar menu stuttering on page load and tooltips not showing when hovering reduced menu

- Dropped Alpine persist plugin
- Removed unnecessary Alpine directives
- Added vanilla JS to manage side menu and dark mode through local storage
- Added CSS classes for the above changes

#442

### CODE REVIEW

File: packages/admin/resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -22,22 +22,49 @@ class="h-full">
...
+            display: none !important;
+        }
     </style>
 
-    <script defer
-            src="https://unpkg.com/@alpinejs/persist@3.x.x/dist/cdn.min.js"></script>
     <script defer
             src="https://unpkg.com/alpinejs@3.8.1/dist/cdn.min.js"></script>
+
+    <script>
```
Comment (alecritson) [inline packages/admin/resources/views/layouts/app.blade.php:38]: Looks like quite a lot is happening here and there's some repetition in the overall logic. Could this be added to the `hub.js` file so it's compiled and can be reused?

**Comment by markmead** [comment]
I don't think we should be straying too far from the current implementation, we just need a way to handle the side menu `width` before Alpine JS initializes. After that Alpine JS can manage the rest.

**Comment by alecritson** [comment]
Thanks @skyrowl We're going to go with this pr #446 for the time being, if we need to look at dark mode then we can do that in a later release. Thanks for taking a look though.

File: packages/admin/resources/views/layouts/app.blade.php (hunk 1 above)
Comment (skyrowl) [reply packages/admin/resources/views/layouts/app.blade.php:38]: Not really, it must be it must be executed before the DOM starts loading else the stutter will come back even for a second or less but still visible.
Repetition can be removed tho it is to make sure that the value is either "true" or "false" (localStorage is treated as a string even for booleans) so an error through JSON.parse() can be avoided if the value is a string other than these two values (JSON.parse can't be used with a literal string that is not a bool).

Comment (markmead) [reply packages/admin/resources/views/layouts/app.blade.php:38]: Don't need to worry about passing wrong data to `JSON.parse` as Alpine JS will always be sending either `true` or `false` and as you say, local storage treats it as a string 😄

**Pull request closed without merging**